### PR TITLE
Add Solus logo

### LINKED ISSUE #358: Add Solus logo to the snapcraft homepage
Solus is now a snap supported system (out of the box) so we should add their distro logo to the home page along with the other supported distros.

The branding can be found at https://solus-project.com/branding/

### PR DESCRIPTION
**Pull request by caldav**
## Done

Added Solus logo to homepage with link to install doc (merged, not deployed)

## Issue / Card

Fixes #358

## Screenshots

screenshot from 2017-08-16 18-10-50

### CODE REVIEW

**Comment by yaili** [comment]
@kwm14 agree, there used to be a page which had lots of logos in the hero and they were inside cards, which made it look neater

**Comment by caldav** [comment]
@kwm14 fully agreed that it needs a change. 4-4-2 seems a bit too wide to accommodate the content on the left. What about a full width 5-5 underneath the text content (that could be widened and centered)?

@yaili each logo inside a card?

**Comment by caldav** [comment]
Playing a bit with icon size, I think I arrived to an acceptable state with 4-4-2.

screenshot from 2017-08-17 14-26-08

Obviously, this approach won't scale in the future.

**Comment by kwm14** [comment]
@caldav in regards to @yaili comment she was referring to this design: https://pplware.sapo.pt/wp-content/uploads/2014/04/ubuntu_phone_04_thumb.jpg

Also here on Ubuntu on 'Charms available for over 300 services' section: https://www.ubuntu.com/cloud/juju

Reviewing with @yaili we propose going with your idea:
- 5-5 logos in boxes  underneath the text
- headline can span across 8-col
- button and link all in one line
- all text to be aligned left

**Comment by caldav** [comment]
screenshot from 2017-08-21 12-23-29

Update. @barrymcgee @kwm14 I'd appreciate your help to tune it for smaller screen sizes.

**Comment by kwm14** [comment]
@caldav looking good, one more design tweak:

- Add `0.75rem` space either-side of 'or' so call to actions aren't so close to each-other
- In regards to small screen as snapcraft.io is using Vanilla the default breakpoints should adjust content positioning accordingly
- Could you attach a small screen screenshot to this issue?

**Comment by caldav** [comment]
@kwm14 good catch. Added inline style for this case.

* Spacing:

screenshot from 2017-08-21 14-51-20

* Small screen (where there is an issue):

screenshot from 2017-08-21 14-50-45

**Comment by kwm14** [comment]
@caldav spacing looks good now.

- Auto-build needs to be capitalised
- In regards to small screen if you add margin-bottom: 0.75rem to the button it will allow space between the copy
- other than that I think the mobile view looks ok

**Comment by caldav** [comment]
@kwm14 doing that, but I was more thinking of the misaligned logo boxes (eg. Solus and OpenWrt)

File: index.html [hunk 1]
```diff
@@ -3,46 +3,47 @@
...
+                    <li class="inline-logos__item box">
                         <a href="/docs/core/install-oe-yocto"><img class="inline-logos__image" src="{{ site.asset_path }}d243274d-openembedded-logo.svg" alt="OpenEmbedded" /></a>
                     </li>
-                    <li class="inline-logos__item">
+                    <li class="inline-logos__item box">
                         <a href="/docs/core/install-oe-yocto"><img class="inline-logos__image" src="{{ site.asset_path }}d9a1a359-yocto-project-transp.png" alt="Yocto Project" />
                     </li>
-                    <li class="inline-logos__item last-col">
+                    <li class="inline-logos__item box">
                         <a href="/docs/core/install-openwrt"><img class="inline-logos__image" src="{{ site.asset_path }}ba58e5e3-Openwrt_Logo.svg" alt="OpenWrt" /></a>
```
Comment (nottrobin) [inline index.html]: I think using `{{ site.asset_path }}4af40b54-Openwrt_Logo.svg` instead will fix the alignment of the logos.

**Comment by caldav** [comment]
screenshot from 2017-08-24 16-33-59

Design issues resolved

**Pull request merged**